lib/asciidoctor/js/postscript.rb

```diff
@@ -1,4 +1,3 @@
-require 'asciidoctor/converter'
```

Comment:
Ah, because of where the require was done for the html5 converter. It was done at the top instead of at the bottom. Now I get it.